### PR TITLE
Added Authentication Page

### PR DESCRIPTION
**Pull request by Ankur9669**
1. Added Login and signup page.
2. Added Links to navbar

#Login Page: 

Preview: https://deploy-preview-5--dazzling-keller-205e9b.netlify.app/login.html

image

#Signup Page

Preview: https://deploy-preview-5--dazzling-keller-205e9b.netlify.app/signup.html

image

### CODE REVIEW

File: authentication.css [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+    margin: 1.8rem 0rem;
+}
+.form-input-box input{
+    width: 90%;
+    padding-left: 1rem;
+    height: 3.12rem;
+    box-shadow: var(--tertiary-color) 0px 1px 8px;
+    border-radius: 3px;
+    background-color: var(--primary-text-color);
+    border: 1px solid #BCBCBC;
```
Comment (yaswanthmyneni) [inline authentication.css:77]: changes: use CSS variable for colors.

File: authentication.css [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+    position: absolute;
+    width: 30%;
+    z-index: -1;
+}
+.blob1{
+    top: -10px;
+    left: -100px;
+}
+.blob2{
+    bottom: 100px;
```
Comment (mehulsatardekar) [inline authentication.css]: use rems for sizing if it is more than 10px

**Review by yaswanthmyneni — APPROVED** [review]
The UI looks great, you can merge it.

**Review by mehulsatardekar — APPROVED** [review]
a suggestion @Ankur9669 on the signup page make the button text of **already have an account** to the white or something for better contrast. The rest looks awesome, coming to ui part it looks neat and elegant.

File: authentication.css (hunk 2 above)
Comment (Ankur9669) [reply authentication.css]: It is not for sizing it is for placement, and if i will use rems it would depend on fontsize

**Review by Naman-Saxena1 — CHANGES_REQUESTED** [review]
You can add labels to input fields for accessibility and then hide them if you don't want them in UI.
Code, commit messages, contrast ratio and other things looks great 👍

**Pull request merged**